src/instructlab/model/train.py

```diff
@@ -311,13 +311,13 @@ def train(
     if four_bit_quant and device != "cuda":
         ctx.fail("'--4-bit-quant' option requires '--device=cuda'")
 
-    effective_data_dir = Path(DEFAULTS.DATASETS_DIR)
+    effective_data_dir = Path(data_path or DEFAULTS.TAXONOMY_DIR)
```

Comment:
Are we still going to output the generated files into the taxonomy directory?

Reply:
Has this been addressed?

Reply:
Yeah, I moved it back to be DATASETS_DIR but kept the conditional in the case when `data_path` is provided.